src/emailservice/email_server.py

```diff
@@ -94,7 +68,7 @@ def Watch(self, request, context, send_response_callback=None):
 
 def start():
   server = grpc.server(futures.ThreadPoolExecutor(max_workers=10),
-                       interceptors=tuple())
+                        interceptors=[server_interceptor()])
```

Comment:
I'd try without this first and see if the service gets auto-instrumented.

Reply:
ack, I did try it initially and it didn't work. But worth revisiting as I might've missed something.

Reply:
thank you! it does work ! removing the interceptors.